closed/get_j9_source.sh

```diff
@@ -173,6 +208,38 @@ if [ ${pflag} = true ] ; then
 	wait
 fi
 
+# Download OCK binaries and create Java module folder.
+openjceplus_source=OpenJCEPlus
+if [ -n "${git_urls[$openjceplus_source]}" ] ; then
+
+	echo
+	echo "$openjceplus_source exists, download OCK binaries"
+	echo
+
+	cd $openjceplus_source
+	mkdir -p ock/jgsk_sdk/lib64
+
+	if [ -n "$gskit_credential" ] ; then
+		# Download GSKit binaries from artifactory.
```

Comment:
The reference to "artifactory" isn't necessarily appropriate; the URLs may point elsewhere.
```
		# Download GSKit binaries.
```
or just remove the comment.